### ISSUE TITLE
Reinstate example projects

### ISSUE THREAD

**Issue by rossabaker** [http4s/http4s#2107]
We urgently need to get those cleaned up and compiling again.

- Should build on what we learned in http4s/http4s.g8
- Should use less inheritance and look more like independent projects so people see how it fits together.

**Comment by calvinbrown085** [comment]
@rossabaker Where are these example projects? I don't see them https://http4s.org/v0.18/ I might be missing something though

**Comment by calvinbrown085** [comment]
Ah, maybe they are here? https://github.com/http4s/http4s/tree/f353f2314b945ff14317f1fb4366aea4b7321ebf/examples

**Comment by rossabaker** [comment]
Yes, those are the ones.

**Comment by rossabaker** [comment]
Before we duplicate work, I think @aeons mentioned on Gitter he was looking at these.  Maybe they can be split up, maybe not.

**Comment by aeons** [comment]
Yeah, I started looking at them, but haven't gotten very far yet.
